src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -144,12 +145,9 @@ struct miopen_convolution
 #endif
     }
 
-    inline void set_conv_descriptor()
+    void set_conv_descriptor()
```

Comment:
Why remove inline from here?

Reply:
I'm not entirely sure why @pfultz2 did this, but maybe because changing the inline function added significant overhead to rebuilding or caused compatibility issues.

Reply:
All good. I just wasn't sure why we would remove that. Its minor enough that I dont think it breaks things.